Readme.md

```diff
@@ -217,3 +229,17 @@ List of user folders:
 Custom textures have to be placed in the user directory under
 `Load/Textures/[GameID]/`. You can find the Game ID by right-clicking a game
 in the ISO list and selecting "ISO Properties".
+
+## Downloading Pull Request (PR) Builds
```

Comment:
I'm against this section. readme.md AFAIK tells users how to use (and compile) Dolphin.

How to test PRs is out of scope of Dolphin and honestly, I don't like the idea of making it any easier for clueless users who don't know how to test PRs, testing them and creating noise due to bad testing.

Please remove it.

Reply:
All of the modifications I proposed are not essential to using and compiling Dolphin but neither is the Custom Textures section. However, like the Custom Textures section, it is good and non-intuitive information for users and devs to have.

Users and potential future devs who find the somewhat-hidden Github page and take the time to read the documentation would likely want to know about testing PR builds and the other information in this PR. It is likely that this will increase the amount of good and bad testers (hopefully more of the former).

I'm open to moving the "Downloading Pull Request (PR) Builds" section to somewhere else but I feel it should be more accessible information than it already is. If you still feel that it should be removed completely, I'd like a third opinion on the latest version of this PR.